src/fsharp/FSharp.Core/array.fs

```diff
@@ -120,8 +121,8 @@ namespace Microsoft.FSharp.Collections
             
         [<CompiledName("Replicate")>]
         let replicate count x = 
-            if count < 0 then invalidArg "count" (SR.GetString(SR.inputMustBeNonNegative))
-            let arr = (Microsoft.FSharp.Primitives.Basics.Array.zeroCreateUnchecked count : 'T array)  
+            if count < 0 then invalidArg "count" LanguagePrimitives.ErrorStrings.InputMustBeNonNegativeString
```

Comment:
I guess I can revert that.

On Jul 22, 2016 11:39 AM, "Don Syme" <notifications@github.com> wrote:

> In src/fsharp/FSharp.Core/array.fs
> <https://github.com/Microsoft/visualfsharp/pull/1347#discussion_r71852825>
> :
>
> > @@ -120,8 +121,8 @@ namespace Microsoft.FSharp.Collections
> >
> >          [<CompiledName("Replicate")>]
> >          let replicate count x =
> > -            if count < 0 then invalidArg "count" (SR.GetString(SR.inputMustBeNonNegative))
> > -            let arr = (Microsoft.FSharp.Primitives.Basics.Array.zeroCreateUnchecked count : 'T array)
> > +            if count < 0 then invalidArg "count" LanguagePrimitives.ErrorStrings.InputMustBeNonNegativeString
>
> Actually IIRC it's hard to fix - amking those "inlined" values into
> functions involves adding surface area to FSharp.Core. There was some
> reason we pulled out of fixing it though I'm not sure why.
>
> Is there any particular reason you want to use the direct string?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Microsoft/visualfsharp/pull/1347/files/057f9aa43438c2f76a1505ddb288de1887437824#r71852825>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADgNEMFoQrpWeBPeW2UZ7Ti1HLP2Z3Tks5qYI_ggaJpZM4JLdjM>
> .
>